### PR TITLE
add fields that are already in the exim-routers

### PR DESCRIPTION
**Pull request by Udera**
the spam_drop-field is used here: https://github.com/avleen/vexim2/blob/master/docs/debian-conf.d/router/249_vexim_ditch_routers#L49
even it didn't exist before!

### CODE REVIEW

File: setup/mysql.sql [hunk 1]
```diff
@@ -59,6 +59,7 @@ CREATE TABLE IF NOT EXISTS `vexim`.`users`
 	on_piped         bool                    NOT NULL  default '0',
 	on_spamassassin  bool                    NOT NULL  default '0',
 	on_vacation      bool                    NOT NULL  default '0',
+	spam_drop		bool                    NOT NULL  default '0',
```
Comment (rimas-kudelis) [inline setup/mysql.sql]: can you please fix the alignment here?

File: setup/create_db.pl [hunk 2]
```diff
@@ -221,6 +221,7 @@ sub create_mysqltables {
 	on_piped bool NOT NULL default '0',
 	on_spamassassin bool NOT NULL default '0',
 	on_vacation bool NOT NULL default '0',
+	spam_drop bool NOT NULL default '0',
```
Comment (rimas-kudelis) [inline setup/create_db.pl]: Can you add this field to the commented out block in `create_postgrestables` (see Line 330)?

**Comment by Udera** [comment]
@rimas-kudelis thanks, I missed the commented part.

**Comment by rimas-kudelis** [comment]
It seems we also don't have the UI which would set this flag. Could you please add that as well?

**Comment by Udera** [comment]
I have opened an issue here (https://github.com/avleen/vexim2/issues/44) that we don't forget to fix this. Either I put this in a new pull request or if you don't merge it yet, I will attach it to this one.

File: setup/mysql.sql [hunk 3]
```diff
@@ -59,6 +59,7 @@ CREATE TABLE IF NOT EXISTS `vexim`.`users`
 	on_piped         bool                    NOT NULL  default '0',
 	on_spamassassin  bool                    NOT NULL  default '0',
 	on_vacation      bool                    NOT NULL  default '0',
+	spam_drop	 bool                    NOT NULL  default '0',
```
Comment (rimas-kudelis) [inline setup/mysql.sql]: This still should use spaces, except at the beginning of the line.

**Comment by Udera** [comment]
I added the spam_drop function to the web-interface.

File: vexim/userchange.php [hunk 4]
```diff
@@ -47,60 +47,119 @@
...
+          ?>
+  	  <tr>
+  	    <td><?php echo _('Maximum message size'); ?>:</td>
+  	    <td>
+            <input type="text" size="5" name="maxmsgsize"
+              value="<?php echo $row['maxmsgsize']; ?>" class="textfield">Kb
+	    </td>
+  	  </tr>
+  	  <tr>
+  	    <td><?php echo _('Vacation on'); ?>:</td>
```
Comment (rimas-kudelis) [inline vexim/userchange.php]: I think "Vacation enabled" was a better message.

File: vexim/userchange.php [hunk 5]
```diff
@@ -47,60 +47,119 @@
...
+	    </td>
+  	  </tr>
+  	  <tr>
+        <?php if (function_exists('imap_qprint')) { ?>
+  	    <td><?php echo _('Vacation message'); ?>:</td>
+  	    <td>
+            <textarea name="vacation" cols="40" rows="5" class="textfield"><?php print imap_qprint($row['vacation']); ?></textarea>
+          </td>
+        <?php } else { ?>
+  	    <td><?php echo _('Vacation message (ASCII only!)'); ?>:</td>
```
Comment (rimas-kudelis) [inline vexim/userchange.php]: This could probably say "ASCII only or QP-encoded!"

**Comment by rimas-kudelis** [comment]
Thanks @Udera!
Next time please avoid big cosmetic changes in unrelated pull requests. They make it more difficult to read a diff.
I left two comments on the strings, but I'm merging your PR in.

**Comment by Udera** [comment]
@rimas-kudelis I first made the changes to adminuseradd.php and the style there is different. I didn't want to rewrite everything. And the style with putting all html-code into print-commands is not very readable either.

**Comment by rimas-kudelis** [comment]
Well, I can't disagree on that. This file was really irregular.

**Comment by rimas-kudelis** [comment]
Oops, a bug slipped in this patch. I've just pushed a fix: df0bfca .

**Pull request merged**